src/legacy/ui/public/new_platform/new_platform.ts

```diff
@@ -59,6 +59,7 @@ export interface PluginsSetup {
   share: SharePluginSetup;
   usageCollection: UsageCollectionSetup;
   advancedSettings: AdvancedSettingsSetup;
```

Comment:
should also update the setup/start mocks over in `src/legacy/ui/public/new_platform/__mocks__/helpers.ts` to ensure the advanced settings mocks are pulled in if someone does `jest.mock('ui/new_platform')`